app/views/patients/update.js.erb

```diff
@@ -36,3 +36,14 @@ $('#change_log').html(
   // clear out fulfillment form
   $('#fulfillment').empty();
 <% end %>
+
+// flash success
+if ($( "#flash" ).length == 0) {
+  $('main > .container').prepend("<div id='flash'></div>")
+}
+
+$('#flash').removeClass().hide().fadeIn('slow')
+$('#flash').addClass("alert alert-success")
+$('#flash').html(
+  '<button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button><div id="flash_notice"> Patient data saved!</div>'
```

Comment:
It'd be in the update.js -- pretty much a drop in replacement for `'button type="button".....` as appears here.

From futzing, here's some drop in code to try out:

```
# patients controller
    if @patient.update_attributes patient_params
      @patient.reload
      flash.now[:notice] = 'Patient info successfully saved' # <- this saves the flash into a session variable that gets cleared out on next page load
      respond_to { |format| format.js }
# ...etc

# the flash success section of update.js.erb
$('#flash').html('<%= escape_javascript(render partial: "layouts/messages") %>'); // <- this just lets the existing template do the lifting of what to render for us
$('#flash').removeClass().hide().fadeIn('slow')
```

ps in futzing around with this I tried out the `fadeIn` effect and I really, really, really like it.